addons/mass_mailing_sms/views/mass_mailing_sms_templates_portal.xml

```diff
@@ -12,7 +12,7 @@
                             <div class="form-group row">
                                 <label for="sms_number" class="col-sm-2 col-form-label">Number</label>
                                 <div class="col-sm-10">
-                                    <input type="text" class="form-control" name="sms_number" id="sms_number"/>
+                                    <input type="text" class="form-control" name="sms_number" id="sms_number" t-att-required="true"/>
```

Comment:
A bit out of scope but ok

Reply:
Is was just weird to get a successful blacklist message as we haven't give any number...
Do I remove it?

Reply:
no, it's fine